DevelopGitHub/GitHubExtension/GitHubExtension.Entities/Class1.cs

```diff
@@ -1,13 +0,0 @@
-﻿
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace GitHubExtension.Entities
```

Comment:
Sorry file was removed from solution but it left in project folder. We are fixing it now and send a new pull request